compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -1157,6 +1157,15 @@ unsafe extern "C" {
     pub(crate) fn LLVMGetNamedGlobal(M: &Module, Name: *const c_char) -> Option<&Value>;
     pub(crate) fn LLVMGetFirstGlobal(M: &Module) -> Option<&Value>;
     pub(crate) fn LLVMGetNextGlobal(GlobalVar: &Value) -> Option<&Value>;
+
+    pub(crate) fn LLVMGetFirstFunction(M: &Module) -> Option<&Value>;
```

Comment:
I made up the convention of only introducing new functions to enzyme_ffi.rs if it's only used by Enzyme.
Only if I find users outside of our own code (or someone else start using it) I put it here in ffi.rs. Not super relevant, but do you mind moving it? That makes it a bit easier to track.